### PR TITLE
Add Shelly

### LINKED ISSUE #4633: Shelly
**Name:** Shelly
**Website:** https://shelly.cloud
**Alexa rank:** 90,279

Regards Marc

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Shelly

**Issue:** Resolves #4633
**Alexa rank:** [~105k](https://www.alexa.com/siteinfo/shelly.cloud)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Pleasantly surprised with how this one turned out! I recreated the encircled "S" they use for their favicon and on all their social media sites by taking the "S" from the logo on the first page in [this PDF](https://shelly.cloud/documents/catalogues/catalogue.pdf) (linked from [website](https://shelly.cloud/) footer) and lining it up as best I could with the one in [this PNG](https://shelly.cloud/wp-content/uploads/2020/05/cropped-shelly_fav-192x192.png). It needed to be stretched a bit so may not line up _exactly_. Colour is from the same PDF.

### CODE REVIEW

**Review by adamrusted — APPROVED** [review]
Spot on! Thanks @PeterShaggyNoble 🚀
Given you've manually vectorized this (albeit just the circle and positioning the S) I'll wait for another reviewer before merging this.

**Review by service-paradis — APPROVED** [review]
LGTM 💯 
Good job @PeterShaggyNoble!!

**Pull request merged**
